fr/graph-views/secure-your-map-platform.md

```diff
@@ -213,6 +213,20 @@ RUN_ARGS="--spring.profiles.active=prod,tls_broker"
 > "tls_broker" profile implies "tls" profile. So Centreon MAP service
 > serves necessarily HTTPS.
 
+Once you add a truststore, Centreon will only use the truststore to validate certificates.
```

Comment:
Once you add a truststore, Centreon Map will be based on it to validate self-signed certificates.